README.md

```diff
@@ -327,6 +328,12 @@ For example, with the following setting, only `if` and `switch` tokens are check
 </rule>
 ```
 
+#### SlevomatCodingStandard.ControlStructures.JumpStatementsSpacing 🔧
+
+Enforces configurable number of lines around jump statements (continue, return, ...).
+
+See `SlevomatCodingStandard.ControlStructures.BlockControlStructureSpacing` for more information (same parameters apply).
```

Comment:
Please copy the options sections. And the new option `allowSingleLineYieldStacking` :)